interview-150/80.ts

```diff
@@ -0,0 +1,19 @@
+function removeDuplicatesII(nums: number[]): number {
+    let numOcc = 1
```

Comment:
Variable name here could be a bit clearer - assuming this means "number of occurences", even so this is a little wonky because it starts at 1 and you count down to 0.

A better approach might be to count _up_ to make the intention more clear